index.js

```diff
@@ -35,7 +35,10 @@ function makeCdnizer(opts) {
 						package: fileInfo.package,
 						test: fileInfo.test
 					});
-					result += _.template(fileInfo.cdn || opts.defaultCDN, lodashTemplateSettings)(params);
+					var excludeCdnPrefix = url.indexOf('http') >= 0 || url.indexOf('//') === 0;
```

Comment:
This check should include the colon, so we don't accidentally match something that starts with `http` but isn't a URL.

I'd use a regex, like so:

```js
var excludeCdnPrefix = /^(https?:)?\/\//.test(url);
```

Alternatively, it could exclude all absolute URLs, no matter the protocol, like so:

```js
var excludeCdnPrefix = /^([a-z]+:)?\/\//i.test(url);
```